addon-test-support/-private/setup-window-mock.js

```diff
@@ -7,5 +7,5 @@ import { reset } from 'ember-window-mock';
 //
 export default function setupWindowMock(hooks = self) {
   // @TODO: assert that window mock is pristine in `hooks.before`.
-  hooks.afterEach(reset);
+  hooks.afterEach(() => reset());
```

Comment:
Since `reset` now accepts an optional proxy, it needs to be invoked like this.

This is also why I needed to remove `tests/unit/setup-window-mock-test.js`. But since we test that `setupWindowMock` does indeed reset the mock in `tests/integration/window-test.js`, I think we're in the clear.

Reply:
`afterEach` and all other hooks receive `assert` as an argument. `reset` takes an optional argument, a proxy, to only reset that proxy.

This means you could call `reset(window)` which is equivalent to `reset()`, but also `reset(window.navigator)` to _only_ reset the `window.navigator` proxy. This features is not strictly required, and if you prefer I can remove it. 🙃

Reply:
> afterEach and all other hooks receive assert as an argument. 

Ah, of course! 🤦‍♂️
I got too used to mocha, which we usually use :wink:

> This features is not strictly required, and if you prefer I can remove it. 

No, I think it adds value, so let's keep it. Hopefully `setupWindowMock()` is primarily used, so that it is not a big problem.